doc/index.rst

```diff
@@ -28,11 +28,14 @@ Here is an example of using the Altair API to quickly visualize a dataset:
 
 .. altair-plot::
 
-    from altair import Chart, load_dataset
+    from altair import Chart, load_dataset, enable_mime_rendering
 
     # load built-in dataset as a pandas DataFrame
     cars = load_dataset('cars')
 
+    # Needed for rendering in JupyterLab, nteract, omit in the classic Jupyter Notebook
+    enable_mime_rendering()
```

Comment:
Since this is an irreversible operation, we might comment it out for people who do a mass copy-paste and run the code before reading it.